common/services/frameworks.js

```diff
@@ -58,13 +58,28 @@ function transformQuestion(
   }
 
   if (options) {
-    field.items = options.map(({ value, label, followup }) => {
-      return {
-        value,
-        text: label,
-        conditional: followup,
+    field.items = options.map(
+      ({ value, label, followup, followup_comment: followupComment }) => {
+        const commentField = {
+          rows: 4,
```

Comment:
Will the `rows` and classes values always be the same?

Reply:
Yes, for this "field" the intention is to always display them in the same manor. Basically the desire is to be able to capture "more details" about a particular yes/no question without having to create a new question within the framework and send it as a separate response.

In future some of these may be replaced with more specific fields to try and capture better data from users.